app/controllers/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.create(user_params)
```

Comment:
don't forget to control for if a user can't be saved in the database - what if they don't enter in all the required fields?